### PR TITLE
Add Author Bloc, and rename button/link stories to index.stories.js

### PR DESCRIPTION
**Pull request by JoffreyMaheo**

### CODE REVIEW

File: src/components/Author/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+Author.propTypes = {
+  name: PropTypes.string.isRequired,
+  time: PropTypes.string.isRequired,
+  avatar: PropTypes.string,
+};
+
+Author.defaultProps = {
+  avatar:
+    'data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjAgMjMiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+DQogICAgPGcgZmlsbD0iI0I5QkJDQSIgZmlsbFJ1bGU9ImV2ZW5vZGQiPg0KICAgICAgPHBhdGggZD0iTTEwLjMzNSA2LjY1Nkg1LjM0NlYuOTZjMC0uNDA2LjMzLS43MzcuNzMzLS43MzdoMTMuMDFjLjQwNCAwIC43MzIuMzMzLjczMi43NHYzLjE5MmMwIC40NC0uMjczLjc1NS0uNzA3Ljc1NWgtOC43OHYxLjc0NnpNNS4xNjkgMjIuOTA2bDUuMTY2LTQuNjg1aDguNzY0Yy40MiAwIC43MjQuMzYuNzI0LjczdjMuMjE5YzAgLjQwNi0uMjkyLjczNi0uNjk2LjczNkg1LjE3ek0xMC4zMzUgMTMuOTk3bDYuNjItMi44MzZoLTYuNjJWNi42NTZMNi40OTkgMTEuMTZIMGw0LjE1IDIuODM2TC41ODYgMTguMjJsNS45NTEtMi41OTQgMy43OTcgMi41OTR6IiAvPg0KICAgIDwvZz4NCiAgPC9zdmc+',
```
Comment (bdo) [inline src/components/Author/index.js]: Why didn't you use the new way to add SVGs? See the [Adding SVGs section of CONTRIBUTING.md](https://github.com/EurosportDigital/web-toolkit/blob/master/CONTRIBUTING.md#adding-svgs)

File: src/components/Author/index.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    <StyledSpacer />
+
+    <StyledContent>
+      <StyledName>{name}</StyledName>
+      <StyledTime>{time}</StyledTime>
+    </StyledContent>
+  </StyledAuthor>
+);
+
+Author.displayName = 'Author';
```
Comment (bdo) [inline src/components/Author/index.js]: Are you sure you need this? Could you please try without and see if it makes any difference in the target app by linking to the toolkit?

File: src/components/Author/index.test.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { shallow } from 'enzyme/build';
+import React from 'react';
+import Author from '.';
+
+describe('Author', () => {
+  it('renders Author', () => {
+    expect(shallow(<Author name="name" avatar="avatar" time="time" />)).toMatchSnapshot();
+  });
+  it('renders Author without avatar', () => {
+    expect(shallow(<Author name="name" time="time" />)).toMatchSnapshot();
```
Comment (bdo) [inline src/components/Author/index.test.js]: I think the test would be more readable if the expectation would check that the author's avatar defaults to the default avatar without having to go to the snapshot and then to the source to understand that.

File: src/components/Author/index.js (hunk 2 above)
Comment (JoffreyMaheo) [reply src/components/Author/index.js]: It's seem's working without that now :)

File: src/components/Author/index.test.js (hunk 3 above)
Comment (JoffreyMaheo) [reply src/components/Author/index.test.js]: Done !

File: src/components/Author/index.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+import E from '../../assets/channels/E.svg';
+import { colors, theme, breakpoints } from '../..';
+
+const StyledAuthor = styled.div`
+  display: flex;
+  align-items: center;
+`;
+const StyledAvatar = styled.div`
```
Comment (ldabiralai) [inline src/components/Author/index.js]: Should this be a div? Why not an `img` with an alt tag of the author name?

File: src/components/Author/index.js [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      <StyledName>{name}</StyledName>
+      <StyledTime>{time}</StyledTime>
+    </StyledContent>
+  </StyledAuthor>
+);
+
+Author.propTypes = {
+  name: PropTypes.string.isRequired,
+  time: PropTypes.string.isRequired,
+  avatar: PropTypes.string,
```
Comment (ldabiralai) [inline src/components/Author/index.js]: We're introducing another prop for images here. We have `img` in `<Card />` and `picture` in https://github.com/EurosportDigital/web-toolkit/pull/33 and now `avatar` in this PR.

File: src/components/Author/index.js [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+const StyledName = styled.p`
+  color: ${colors.white};
+  font-weight: bold;
+`;
+const StyledTime = styled.p`
+  color: ${colors.manatee};
+`;
+
+const Author = ({ name, avatar, time }) => (
+  <StyledAuthor>
```
Comment (ldabiralai) [inline src/components/Author/index.js]: We need to spread other props down here and test.

https://github.com/EurosportDigital/web-toolkit/blob/master/CONTRIBUTING.md#requirements
> Components should spread props onto the top level component, for example: const Comp = ({...props}) => <div {...props} />, this should have a test

File: src/components/Author/index.js (hunk 4 above)
Comment (JoffreyMaheo) [reply src/components/Author/index.js]: yes you are right, Done !

File: src/components/Author/index.js (hunk 5 above)
Comment (JoffreyMaheo) [reply src/components/Author/index.js]: Done !

File: src/components/Author/index.js (hunk 6 above)
Comment (JoffreyMaheo) [reply src/components/Author/index.js]: Done !

**Comment by bdo** [comment]
Closing as superseded by #47

**Pull request closed without merging**